src/main/java/seedu/contax/logic/commands/FindAppointmentCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.contax.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.contax.commons.core.GuiListContentType;
+import seedu.contax.commons.core.Messages;
+import seedu.contax.model.Model;
+import seedu.contax.model.appointment.ContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all appointments in address book whose name contains any of the argument keywords.
+ * Keyword matching is case-insensitive.
+ */
+public class FindAppointmentCommand extends Command {
+    public static final String COMMAND_WORD = "findappointment";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
```

Comment:
update the params and examples as well

Reply:
Fixed

Reply:
can add the possibility for findappointment [text] by/person also as the params, like optional params

Reply:
Alright added